.github/workflows/release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       # https://code.visualstudio.com/api/working-with-extensions/publishing-extension
       - name: Install VSCE
         run: |
-          npm install -g "vsce@$(jq -r '.dependencies.vsce.version' < packages/vscode-pyright/package-lock.json)"
+          npm install -g npm install -g "@vscode/vsce@$(jq -r '.dependencies["@vscode/vsce"].version' < packages/vscode-pyright/package-lock.json)"
```

Comment:
I'm guessing  'npm install' being twice is a typo?

Reply:
yes it is!